docs/user/configuration-management.rst

```diff
@@ -336,7 +335,7 @@ Setting the statistics (and cache) parameters is necessary when the local redis
 Runtime Configuration
 *********************
 
-This configuration is used by each bot to load its specific (runtime) parameters. Usually, the ``BOTS`` file is used to generate ``runtime.conf``. Also, the IntelMQ Manager generates this configuration. You may edit it manually as well. Be sure to re-load the bot (see the :doc:`intelmqctl`).
+This configuration is used by each bot to load its specific (runtime) parameters. You can use the output of ``intelmqctl list bots`` to write the ``runtime.conf``. Also, the IntelMQ Manager generates this configuration. You may edit it manually as well. Be sure to re-load the bot (see the :doc:`intelmqctl`).
```

Comment:
`intelmqctl -t josn list bots`? Actually I'm missing a reference to the feeds docs here, but that could be out of scope here. (and a reference to `contrib/feeds-config-generator` could be added as well).

Reply:
fixed. not sure what reference you're referring to or what you have in mind, but if you create a separate issue, i'll have a look